tlsfuzzer/helpers.py

```diff
@@ -307,3 +307,22 @@ def __new__(cls):
         if not hasattr(cls, 'instance') or not cls.instance:
             cls.instance = object.__new__(cls)
         return cls.instance
+
+
+def protocol_name_to_tuple(name):
+    names = {"sslv2": (0, 2),
+             "ssl2": (0, 2),
+             "sslv3": (3, 0),
+             "ssl3": (3, 0),
```

Comment:
it looks like a discrepancy that only ssl3 is accepted while tls1.0 requires the `.0`. I'd have expected ssl3.0 instead.

Reply:
I've used just the most common names as of now, SSL 3.0 definitely isn't one

are you suggesting I should add "tls1" and "tlsv1" as options?

Reply:
I'm not sure about the definition of commonness, but I've always considered SSL3.0 to be the common name. Anyway, if these are listed somehow to the user I think that's fine as it is now.

Reply:
yes, they are listed in the test scripts that use this method